compiler/compiler.go

```diff
@@ -286,7 +293,14 @@ func (c *Compiler) Compile(node ast.Node) error {
 		if err != nil {
 			return err
 		}
-		c.emit(code.OpCall)
+
+		for _, a := range node.Arguments {
+			err := c.Compile(a)
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
### Calling convention

In vm, put `compiledFunction` → `argument`s and then it will read `Opcall` instructions.